index.bs

```diff
@@ -3093,6 +3093,42 @@ This [=registration extension=] and [=authentication extension=] enables use of
                     01           -- Subitem 3: CBOR short for Matcher Protection Type Software
     </pre>
 
+## Authenticator Trust Anchor Extension (trustAnchor) ## {#sctn-authenticator-trust-anchor-extension}
+
+This [=registration extension=] allows a [=[RP]=] to specify the acceptable trust anchors (attestation formats)
+to be used when creating the credential. 
+
+: Extension identifier
+:: `trustAnchor`
+
+: Client extension input
+:: Acceptable Trust Anchors:
+
+    <pre class="idl">
+    dictionary biometricCriteria{
+        boolean packed;
+        boolean tpm;
+        boolean androidKey;
+        boolean androidSafetyNet;
+        boolean u2f;
+        };
```

Comment:
the above are not "trust anchors", they seem to be referring to [attestation statement format identifiers](https://w3c.github.io/webauthn/#sctn-attstn-fmt-ids)?

Reply:
See https://github.com/w3c/webauthn/issues/461.  But I am open to changing "trust anchors" to something like "attestation formats".

Reply:
@gmandyam you mean "attestation statement format identifiers" yes?